session/session_test.go

```diff
@@ -34,6 +37,55 @@ func (s *SessionTestSuite) TestAll() {
 	s.Equal("value2", all["key2"])
 }
 
+func (s *SessionTestSuite) TestExists() {
+	s.session.Put("foo", "bar")
+	s.True(s.session.Exists("foo"))
+	s.session.Put("baz", nil)
+	s.False(s.session.Has("baz"))
+	s.True(s.session.Exists("baz"))
```

Comment:
It's `false` when `Has`, but it's `true` when `Exists`, a bit confused.

Reply:
That's just how we define the `Has` and `Exists` methods.

![image](https://github.com/goravel/framework/assets/84431594/24020c93-7ce1-4fbf-a83c-79ae88dc31b9)

Reply:
Makes sense